.github/workflows/build.yml

```diff
@@ -40,6 +44,8 @@ jobs:
        run: |
           chmod +x build.sh
           $GITHUB_WORKSPACE//build.sh -v
+     - name: Test with Nunit on Mono
+       run: $GITHUB_WORKSPACE//test.sh
```

Comment:
Do we need the `full` argument here, like on line 37 above?

Reply:
The integration tests need more work on non-windows platforms. 
I am planning to add it here: https://github.com/chocolatey/choco/pull/2435